Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -869,15 +886,13 @@ void MainWindow::reloadItem() {
   QAction* sender_action = qobject_cast<QAction*>(sender());
   if(!sender_action) return;
   
-  bool ok;
-  int item_index = sender_action->data().toInt(&ok);
-  QObject* item_object = scene->item(item_index);
-  if(!ok || !item_object || sender_action->data().type() != QVariant::Int) {
+  Scene_item* item = (Scene_item*)sender_action->data().value<void*>();
+  QObject* item_object = item;
+  if(!item || !item_object) {
```

Comment:
I do not understand!

1. `item_object` seems useless: it cannot be zero if `item` is non-zero.